apps/coscrad-frontend/src/components/tags/tag-card.tsx

```diff
@@ -0,0 +1,60 @@
+import { ITagViewModel } from '@coscrad/api-interfaces';
```

Comment:
Was this filename going to change?

Reply:
Maybe

Reply:
I see what you mean now. I have moved the logic from `TagCard` into `TagDetailPresenter`. Note that there probably won't need to be a `full-view` \ `thumbnail` distinction for tag. Only categorizables are viewed in thumbnails right now.